ur_kinematics/src/ur_kinematics/__init__.py

```diff
@@ -0,0 +1 @@
+from ur_kinematics import *
```

Comment:
The import is not required. An empty `__init__.py` file is sufficient.

Reply:
you are right, it is within the same package. The __init__ file can be empty